library/packages/src/modules/PackagesUI.rb

```diff
@@ -774,6 +777,22 @@ def ShowInstallationSummary
       ShowInstallationSummaryMap(@package_summary)
     end
 
+    # Show messages coming from libzypp about installed packages
+    #
+    # This messages are retrieved from libzypp.
+    #
+    # @param [Array] result Result from package commit (as it comes from PkgCommit)
+    def show_update_messages(result)
+      commit_result = ::Packages::CommitResult.from_result(result)
+      if commit_result.update_messages.empty?
```

Comment:
I usually prefer to indent as it makes it easier to read (for me), specially when it's not the first line in the method. But I know that in YaST-land :) is quite usual, so I've no problem for changing it.